### ISSUE TITLE
Identity Generation stalls with error message

### ISSUE THREAD

**Issue by grburgos** [storj-thirdparty/qnap-storagenode-app#20]
I have removed the application and reinstalled the version Storj 1.1.3 for QNAP. I have removed previous installed config files to start fresh. After executing the wizard and reached the "Identity Generation Started" and click to generate a new Identity. The page changes and the spinning icon shows. Then I proceed to click the Finish button. This causes a validation error stating "Please enter a valid Path". Even though I have selected Generate an Identity. I'm running STORJ on an admin account. (Not THE Admin user).

**Comment by rAdziuB** [comment]
I have the same issue with my QNAP TS-453Be. Firmware version: 4.5.4.1741. For the storage path, I used: /share/STORJ
I have a clean installation of STORJ qpkg. I tried run an app on the admin user and on a different user with admin rights but with no luck.

**Comment by paragpadsumbiya** [comment]
Hello @grburgos and @rAdziuB 

Can you please send me the screenshot which you have entered as identity path as it look like some issue with string of the path . For me its working fine on our staging but we will be very happy to help you to resolve out your issues and up and running your STORJ app.

Thanks
Parag Padsumbiya

**Comment by grburgos** [comment]
If I do not enter any path and simply click (the enabled button) "I don't have an Identity" and then continue the final page by clicking on finish I get the error screen below.
![image](https://user-images.githubusercontent.com/6518928/132563147-07d2ed04-d567-4879-9cc7-395b984988a3.png)

![image](https://user-images.githubusercontent.com/6518928/132562707-5bd42bbc-9a0f-414c-8cf6-ef4cbb2653f1.png)

I just want to add if I do select a path "/home" I still get the error message when I follow the steps mentioned.
![image](https://user-images.githubusercontent.com/6518928/132563578-96df8d33-8f56-4e76-9354-b023ca4d19b6.png)

**Comment by paragpadsumbiya** [comment]
Hello @grburgos 

Thanks for the details.

I have checked same with the path "/home" and its working fine for us .. So will you able to share the your QNAP version and firmware version so we can try to check accordingly as its looks like some issue regarding different environment.

Thanks
Parag Padsumbiya

**Comment by grburgos** [comment]
The firmware version is: 4.5.4.1741

**Comment by Ziifkah** [comment]
Hello, I have the same problem here.

QNAP :  TS-451A
Firmware : 4.5.4.1800

Tried many tutorials but the application simply refuses to take the file.

![1](https://user-images.githubusercontent.com/38363321/140197783-8eda8a49-6753-4f64-b849-bca4a2c58ebb.png)
![2](https://user-images.githubusercontent.com/38363321/140197786-fa77c843-6099-4946-8710-2f3f4588ed10.png)

**Comment by anthonyolazabal** [comment]
Hello, 
did you find how to fix the invalid path issue ? I'm migrating my node to a QNAP and trying to import my identity but it says invalid path when I select the folder with my certs.
Thanks for your help
Best regards

**Comment by anthonyolazabal** [comment]
I'm running a fresh install of with the following version 5.0.0.1858 (on TS-251)

**Comment by Marti4321** [comment]
Same issue (Please enter a valid path, for identity file)

HS-251+
5.0.0.1858

**Comment by grburgos** [comment]
I stopped waiting on the QNAP app and manually installed the Container. It takes a bit of work because the container app are frozen thus the constant updating of STORJ version is nuisance. None the less this was faster.

**Comment by aaronjlawson** [comment]
Has there been any solution found to the identity path issue? I am encountering the same unfortunately.

**Comment by rochemedia** [comment]
I have the same problem and get stuck at the same place, regardless of whether I generate an identity and place it on my TS-431K or try to generate it on the NAS (it doesn't seem to generate anything and there's no sign of any container being pulled or ever existing on it actually...)

**Comment by gpaulat** [comment]
Same issue occurs on TS-231P, the " Please enter a valid path" message is shown when selecting the identity path. FW version is 5.0.0.1932

**Comment by rAdziuB** [comment]
I recommend use Docker (Container Station) on QNAP. Storj work's well. I am using it since September. It is waste of time waiting for Storj app dedicated for QNAP.

**Comment by kmozurkewich** [comment]
@rAdziuB has good feedback.  The maintenance of this app is not a priority and it done when other higher-priority tasks are completed.

**Comment by rochemedia** [comment]
I ended up setting it up on an old computer (I also have SCPrime running on it anyway) and set up iscsi targets on the QNAP NAS, but I saw somewhere it isn's as good an option to run storj with network drives -- has anyone understood why that is and if it really is better to use up the QNAP resources to run it instead of on another computer and then connect via iscsi?
